src/gameObjects/map_forest/butterfly.gameObject.js

```diff
@@ -0,0 +1,24 @@
+import { scaleFactor } from '../../constants';
+
+export const butterfly = (k, map, spawnpoints) => {
+  k.loadSprite('butterfly', './forest_sprites.png', {
+        sliceX: 16,
+        sliceY: 16,
+        anims: {
+            'walk-side': { from: 86, to: 89, loop: true, speed: 4 },
+            'idle-side': 86,
+        },
+    });
```

Comment:
I figured out the positions

```javascript

k.loadSprite('butterfly', './forest_sprites.png', {
        sliceX: 16,
        sliceY: 16,
        anims: {
            'walk-side': { from: 90, to: 91, loop: true, speed: 4 },
            'idle-side': 90,
        },
    });
```

Reply:
Fixed this issue, thank you for finding the correct positions.

Reply:
I also just now fixed the issues with the butterfly and making the map accessible to other players from the top of the screen in the city.